code/game/objects/items/weapons/improvised_components.dm

```diff
@@ -29,14 +29,14 @@
 	force_divisor = 0.1
 	thrown_force_divisor = 0.1
 
-/obj/item/weapon/material/butterflyhandle/attackby(obj/item/W as obj, mob/user as mob)
-	if(istype(W,/obj/item/weapon/material/butterflyblade))
-		var/obj/item/weapon/material/butterflyblade/B = W
-		to_chat(user, "You attach the two concealed blade parts.")
-		new /obj/item/weapon/material/butterflyconstruction(user.loc, B.material.name)
-		qdel(W)
-		qdel(src)
-		return
+//obj/item/weapon/material/butterflyhandle/attackby(obj/item/W as obj, mob/user as mob)
+//	if(istype(W,/obj/item/weapon/material/butterflyblade))
+//		var/obj/item/weapon/material/butterflyblade/B = W
+//		to_chat(user, "You attach the two concealed blade parts.")
+//		new /obj/item/weapon/material/butterflyconstruction(user.loc, B.material.name)
+//		qdel(W)
+//		qdel(src)
+//		return
```

Comment:
Don't comment out old dead code unless you plan on actually using it in the future, and maybe not even then. Remove it, along with the items and all.